pkg/minikube/download/preload.go

```diff
@@ -47,6 +47,17 @@ const (
 	PreloadBucket = "minikube-preloaded-volume-tarballs"
 )
 
+// Enumeration for preload existence cache.
+const (
+	preloadUnknown = iota // Value when preload status has not been checked.
+	preloadMissing = iota // Value when preload has been checked and is missing.
```

Comment:
all iota's below the first one are optional, which we normally omit in this codebase, otherwise LGTM

Reply:
Done.